README.CentOS7.md

```diff
@@ -0,0 +1,152 @@
+# LORIS CentOS 7.x Notes
+
+This document contains supplemental details on how to perform a basic CentOS 7.x install of LORIS.
+Note that the main README in LORIS assumes that LORIS is being run on Ubuntu.
+
+This Readme assumes you already understand basic UNIX, MySQL and Apache setup and
+settings. If you're not already comfortable troubleshooting sysadmin issues,
+you should not follow this guide.
+
+For further details on the install process, please see the LORIS GitHub Wiki CentOS Install page.  
+
+# System Requirements - Install dependencies
+
+Default dependencies installed by CentOS 7.x may not meet the version requirements for LORIS deployment or development:
+* MariaDB 10.3 is supported for LORIS 21.* 
+* PHP 7.2 is supported for LORIS 21.*
+
+In addition to the above, the following packages should be installed with `yum` and may also differ from the packages referenced in the main (Ubuntu) [LORIS Readme](./README.md). Detailed command examples are provided below.
+ * Apache2
+ * Node
+ * PHP Composer
```

Comment:
Should this composer be removed here, @kongtiaowang ?

Reply:
no, we need composer here.